docs/source/layouts/layout.erb

```diff
@@ -96,15 +96,17 @@
 
                   <li><a class="" href="/using/managing-minishift.html">Managing Minishift</a></li>
 
-                  <li><a class="" href="/using/addons.html">Minishift Add-ons</a></li>
+                  <li><a class="" href="/using/addons.html">HTTP/HTTPS proxies</a></li>
```

Comment:
@hferentschik oops sorry, copy-paste error. That's weird, my local staging didn't show this as a problem...